veracruz-client/src/veracruz_client.rs

```diff
@@ -186,6 +188,32 @@ impl VeracruzClient {
         policy: Policy,
         policy_hash: String,
     ) -> Result<VeracruzClient, VeracruzClientError>
+    where
+        P1: AsRef<path::Path>,
+        P2: AsRef<path::Path>
+    {
+        Self::with_url_id_policy_and_hash(
+            &policy.veracruz_server_url().to_owned(),
```

Comment:
Is this not converting to a `String` and then immediately converting to a slice?  Can the `.to_owned()` be dropped?